ee/modules/502-delivery/templates/argocd/server/d8-ingress.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.global.modules.publicDomainTemplate }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: argocd
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "argocd")) | nindent 2 }}
+  annotations:
+    web.deckhouse.io/export-name: "ArgoCD"
+    web.deckhouse.io/export-icon: "/public/img/argocd.svg"
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    # Suggestions from ArgoCD
+    # https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#ssl-passthrough-with-cert-manager-and-lets-encrypt
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
```

Comment:
How it will work in the case of `OnlyInURI` or `Disabled` https mode (when SSL is terminated or not terminated somewhere else)?

Reply:
Will re-check